Src/MoneyFox.Ui/Views/Statistics/MonthlyAccountCashFlow/StatisticAccountMonthlyCashflowViewModel.cs

```diff
@@ -56,12 +56,11 @@ private async Task InitAsync()
     {
         Accounts.Clear();
         var accounts = mapper.Map<List<AccountViewModel>>(await Mediator.Send(new GetAccountsQuery()));
+        var combinedCashFlow = accounts.Sum(x => x.CurrentBalance);
```

Comment:
I think this is currently not needed, since we don't show the balance in the picker. At some point it actually would make sense to replace the AccountViewModel by a dedicated VM that only contains the fields used (aka an ID and the Name=

Reply:
You are correct. I removed the current balance calculation.